modules/key-card/test/unit/faq.ts

```diff
@@ -0,0 +1,14 @@
+import { generateFaq } from '../../src/faq';
+
+describe('generateFaq', function () {
```

Comment:
I know that these tests are very rudimentary. If somebody has ideas how I can improve the test(s) in a meaningful way then I'm all ears.